src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue9006.cs

```diff
@@ -18,14 +27,16 @@ public Issue9006(TestDevice testDevice) : base(testDevice)
 		[Category(UITestCategories.Compatibility)]
 		public void ClickingOnTabToPopToRootDoesntBreakNavigation()
```

Comment:
This test is failing on iOS (crashing):
`The app was expected to be running still, investigate as possible crash`
<img width="937" alt="image" src="https://github.com/user-attachments/assets/abe30a10-b10a-4574-8a8a-6142e6d40581">

Reply:
@jsuarezruiz  The CI failure highlights a test case issue: Issue9006 - ClickingOnTabToPopToRootDoesntBreakNavigation. I have run this test locally using both VS Code and the command prompt, and it passed successfully. Could you please review it and advise me on what I may have overlooked to resolve the CI issue?
 
![image](https://github.com/user-attachments/assets/24e394db-4449-4080-a124-17fc33c7e8c2)

Reply:
This is a more detailed stack trace:

```
OpenQA.Selenium.StaleElementReferenceException : Cached elements 'By.xpath: //android.widget.ImageButton[@content-desc='Navigate up']' do not exist in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception

   at OpenQA.Selenium.WebDriver.UnpackAndThrowOnError(Response errorResponse, String commandToExecute)
   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
   at OpenQA.Selenium.Appium.AppiumElement.Execute(String commandName, Dictionary`2 parameters)
   at OpenQA.Selenium.Appium.AppiumElement._GetAttribute(String attributeName)
   at OpenQA.Selenium.Appium.AppiumElement.<>c__DisplayClass21_0.<GetAttribute>b__0()
   at OpenQA.Selenium.Appium.AppiumElement.CacheValue(String key, Func`1 getter)
   at OpenQA.Selenium.Appium.AppiumElement.GetAttribute(String attributeName)
   at UITest.Appium.AppiumTouchActions.ElementToClickablePoint(AppiumElement element) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumTouchActions.cs:line 372
   at UITest.Appium.AppiumTouchActions.<TapElement>g__ProcessException|15_0(<>c__DisplayClass15_0&) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumTouchActions.cs:line 117
   at UITest.Appium.AppiumTouchActions.TapElement(AppiumElement element) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumTouchActions.cs:line 108
   at UITest.Appium.AppiumTouchActions.Tap(IDictionary`2 parameters) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumTouchActions.cs:line 84
   at UITest.Appium.AppiumTouchActions.Execute(String commandName, IDictionary`2 parameters) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumTouchActions.cs:line 52
   at UITest.Appium.AppiumCommandExecutor.Execute(String commandName, IDictionary`2 parameters) in /_/src/TestUtils/src/UITest.Appium/Actions/AppiumCommandExecutor.cs:line 20
   at UITest.Appium.HelperExtensions.Tap(IUIElement element) in /_/src/TestUtils/src/UITest.Appium/HelperExtensions.cs:line 288
   at UITest.Appium.HelperExtensions.Tap(IApp app, IQuery query) in /_/src/TestUtils/src/UITest.Appium/HelperExtensions.cs:line 38
   at UITest.Appium.HelperExtensions.TapBackArrow(IApp app, IQuery query) in /_/src/TestUtils/src/UITest.Appium/HelperExtensions.cs:line 1781
   at UITest.Appium.HelperExtensions.TapBackArrow(IApp app, String customBackButtonIdentifier) in /_/src/TestUtils/src/UITest.Appium/HelperExtensions.cs:line 1771
   at Microsoft.Maui.TestCases.Tests.Issues.Issue9006.ClickingOnTabToPopToRootDoesntBreakNavigation() in /_/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue9006.cs:line 34
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
```

Reply:
@jsuarezruiz  I have committed a fix which addressed the test failure in the CI. Could you please review and provide any feedback or concerns you may have?
<img width="394" alt="image" src="https://github.com/user-attachments/assets/f7368aea-0a2f-48e6-a2bc-79c65b3678a7">